### ISSUE TITLE
Enforce 2FA for browser access

### ISSUE THREAD

**Issue by MaxParadiz** [Pissandshittium/pissandshittium#107]
**Issue**

A browser contains a large amount of very private personal information. This may include the browser's history, saved passwords, authentication cookies, and many other important pieces of information that one would prefer to keep private.

The current security browsers of features are very week, as anyone with access to the unlocked device is able to simply open up the browser and access all of this important information. I believe it is the duty of the developers to ensure that users are protected from these security vulnerabilities. 

**Solution**

In order for the developers to fulfill their duty to guard the security of their users, 2-factor authentication must be strictly enforced when first opening the browser, and every hour after that (in case the user walks away and forgets). To maximize security, the only acceptable 2FA options are via an SMS message OR via a proprietary 2FA app (we can't trust an open source 2FA app as its security can be easily compromised).

**Comment by Aikoyori** [comment]
literally Matrix

**Comment by ssoufi-code** [comment]
I don't think this is a good, idea, I think a better idea, is to open up the camera and require a non-local facial recognition every 15 minutes. Another thing is, If they smile, we should then ban them from the app for 30 minutes. (Normal users shouldn't be happy to use the browser)
